core/src/components/segment/readme.md

```diff
@@ -8,6 +8,20 @@ Their functionality is similar to tabs, where selecting one will deselect all ot
 
 Segments are not scrollable by default. Each segment button has a fixed width, and the width is determined by dividing the number of segment buttons by the screen width. This ensures that each segment button can be displayed on the screen without having to scroll. As a result, some segment buttons with longer labels may get cut off. To avoid this we recommend either using a shorter label or switching to a scrollable segment by setting the `scrollable` property to `true`. This will cause the segment to scroll horizontally, but will allow each segment button to have a variable width.
 
+## Accessibility
+
+### Keyboard Navigation
+
+On recieving focus, `ion-segment` will pass focus to the selected `ion-segment-button` element. The component also has full keyboard support for navigating between `ion-segment-button` elements. The following table details what each key does:
+
+| Key                | Function                                                       |
+| ------------------ | -------------------------------------------------------------- |
+| `ArrowRight`       | Selects the next focusable element.                            |
+| `ArrowLeft`        | Selects the previous focusable element.                        |
+| `Home`             | Selects the first focusable element.                           |
+| `End`              | Selects the last focusable element.                            |
```

Comment:
This is a little confusing because whether or not it "selects" depends on the value of `selectOnFocus`. Maybe we just say that it focuses the element rather than selects it.

Above the table you could note that "By default, keyboard navigation will only focus `ion-segment-button` elements, but you can use the `selectOnFocus` property to ensure that they get selected on focus too."